experimental/generation/generator/packages/library/templates/standard/en-us/dynamiclistEntity.en-us.lu.lg

```diff
@@ -16,11 +16,15 @@ ${operations(`${entity}`, allSynonyms(), false)}
 - IF: ${examples}
     - ${join(foreach(indicesAndValues(examples), enum, `    - ${enum.index}:${EOL()}${synonyms(enum.value)}`), EOL())}
 - ELSE:
-    - 
+    - ${join(foreach(generatedEntityValues(), enum, `    - ${enum}:${EOL()}${synonyms(phrases(enum))}`), EOL())}
+
 
 # allSynonyms()
 - IF: ${examples}
     - ${flattenObject(examples)}
 - ELSE:
-    - 
+    - ${generatedEntityValues()}
+
+# generatedEntityValues
+ - ${[`**${entity}1**`, `**${entity}2**`, `**${entity}3**`]}
```

Comment:
You should use entityValue(null) from the generator.  That way these are only defined in one place.